src/CitizenAction/CitizenActionManager.php

```diff
@@ -84,10 +91,20 @@ public function populateRegistrationWithAdherentsInformations(EventRegistrationC
                 'postalCode' => $adherentFind->getPostalCode(),
                 'cityName' => $adherentFind->getCityName(),
                 'createdAt' => $registration->getCreatedAt(),
+                'administrator' => false,
```

Comment:
why do you not use `'administrator' => $adherent->isAdministratorOf($citizenProject),`?

Reply:
Citizen Project is not accessible in this method.

Reply:
not false, but it can be, no?
```php
function populateRegistrationWithAdherentsInformations(..., CitizenProject $cp)
```

Reply:
What is the advantage to add an argument to the method and do an additional heavy call https://github.com/EnMarche/en-marche.fr/blob/master/src/Entity/Adherent.php#L945 for each iteration?